### DISCUSSION TITLE
大家都怎麼關注重要時事?

### DISCUSSION THREAD

**Discussion by Tchang323** [1101-GS4719-C/Python-Community-GS4719-C#2250] [General]
看新聞對生活非常重要，了解周圍發生的事情能幫助我們保持與世界的聯繫。好奇大家通常是透過哪些平台或方法來關注時事。對我而言，由於不太喜歡閱讀文字，我更傾向於通過YouTube影片新聞來獲取資訊。不過，我也覺得自己可能錯過了不少國際時事。大家有沒有什麼建議或是偏好的方式來看新聞？或者可以分享一下你們通常在哪些時候會看新聞呢？

**Comment by Chattenoire** [comment]
華語的話，你可以看報導者、轉角國際，或是一些香港的海外媒體；
英文的話，如果你想知道世界真正的樣子，不要看WSJ、CNN、BBC，請看routers、GIS reports或是ICIJ。
不喜歡閱讀文字會錯過很多最本質的資訊，因為影片永遠都是壓縮過的資訊，不如文字。

**Comment by wcyeee** [comment]
個人比較喜歡靠近台灣的各種各類生活新聞，會下載聯合網的app然後開啟通知，有時候看到有興趣的會點進去。如果是偏向國際或政治的話會有中央社（不太確定是不是這個名字）！

**Comment by forestdew507** [comment]
我也有類似的困擾XD 有時候真的要刻意去網路上翻時事，比起可能已經被演算訓練的 Google、各種社群，上面留言提到的報導者、中央社、換日線等自媒體 ( ? ) 我想會更適合 ! 

FYI 我身邊剛好有喜歡分享時事的朋友，甚至到會幫忙擷取影片片段 + 在限時動態標記 「就算不點進去看 15 分鐘完整影片，也拜託把我這 1 分鐘的精華看完」，看到這樣的資訊我都會強迫自己停下來哈哈。

**Comment by kzx-creat** [comment]
我是追蹤IG上的台灣青年民主協會、報導者、法白、換日線，有新貼文就會看到，超方便。但最近在練習減少IG使用量，所以可能要習慣其他管道(Youtube，媒體官網之類的)。國外媒體的話我較常接觸 BBC, CNN, The Guardian，不過很少認真整篇看完......。

**Comment by Teddy111802518** [comment]
基本上我是google ig fb 都有 但是重點其實是媒體視讀能力 尤其資訊爆炸的時代 又一堆假新聞 最好再評論前或接受資訊前再求證一下 你的隨便一句話很傷人

**Comment by ed112601513** [comment]
我會從電視、yt、google新聞來關注重要時事，我覺得從哪裡獲取新聞時事並不重要，重要的是自己關注什麼，以及立場有沒有偏頗，畢竟個人喜好會很大程度的影響演算法，而演算法才會影響到自己接收新聞時事的廣泛程度，最後要能判斷假新聞並多方查證，避免自己接收到的訊息是有問題的
